### DISCUSSION TITLE
[FEATURE REQUEST]: Advanced Company AntiKick

### DISCUSSION THREAD

**Discussion by D1GQ** [winstxnhdw/lc-hax#385] [Ideas]
Do you think there's a possible way to make an anti-kick for Advance Company? the current method doesn't work for it.

**Comment by winstxnhdw** [comment]
I took a look at the mod and it seems like they completely reworked the lobby system. It’s not possible for us to add support for that mod while still supporting vanilla.

**Reply by D1GQ** [reply]
Well if it's not possible to have both but it's still possible to make an anti-kick for the mod, maybe there could be a toggle between the two? Inside the actual files for the hack.

**Reply by winstxnhdw** [reply]
Sure, but then I'd have to build against their assembly every time and I don't really want to do that since I am not sure about the general good practices for that, and if it will get me a DMCA.

**Reply by D1GQ** [reply]
Well okay understandable, was just a thought. But this does make me curious though is there any way to spoof your steam ID in the game? vanilla or modded.

**Reply by winstxnhdw** [reply]
If there was a way, we’d be the first to implement it.

**Reply by xAstroBoy** [reply]
Technically you can detect the dll with reflection, i did it in the past and go from there , without adding it as dependency

**Reply by winstxnhdw** [reply]
Sure? But I’d have to reflect every single thing, which honestly sucks. I can’t imagine supporting something that unstable.
